docs/en-US/component/menu.md

```diff
@@ -59,6 +59,7 @@ menu/collapse
 | collapse                | whether the menu is collapsed (available only in vertical mode)                                                                                                       | boolean | —                     | false    |
 | ellipsis                | whether the menu is ellipsis (available only in horizontal mode)                                                                                                      | boolean | —                     | true     |
 | background-color        | background color of Menu (hex format) (deprecated, use `--bg-color` instead)                                                                                          | string  | —                     | #ffffff  |
+| active-bg-color         | background color of currently active menu item (hex format) (deprecated, use `--active-bg-color` instead)                                                             | string  | —                     | —        |
```

Comment:
It may not be a good idea to add a property to be deprecated, and you can modify the style yourself

Reply:
> It may not be a good idea to add a property to be deprecated, and you can modify the style yourself

Thank you for your reply! I understand that it's weird to add a deprecated prop. It's just that I think it'll be more thoughtful to add a property to customize active bg color for the reasons below:

1. There's already an `active-text-color`😂 Users might want to know how to set active-bg-color, so why not? 😄
2. `active-text-color` is deprecated due to `--active-color` variable, but there's no `--active-bg-color` correspondingly (or maybe I did not search for the variable correctly?).
3. If we don't provide this property, developers will have to write CSS manually to override active bg color for menus with specific class names. This might cause redundant codes. If we provide this property, users can customize it easily by adding this property.

These reasons are why I'm introducing this property. Sincerely hope that you can rethink it.🙏🏻